manifests/b/BrianApps/Sizer/Dev/4.0.640/BrianApps.Sizer.Dev.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BrianApps.Sizer.Dev
+PackageVersion: 4.0.640
+PackageLocale: en-US
+Publisher: Brian Apps
+PublisherUrl: http://www.brianapps.net/
+PublisherSupportUrl: http://www.brianapps.net/contact.html
+Author: Brian Apps
+PackageName: Sizer (Dev)
+PackageUrl: http://www.brianapps.net/sizer/
```

Comment:
shouldn't this be http://www.brianapps.net/sizer4/? :)

Reply:
Indeed, thank you 👍